src/neptune_scale/core/components/aggregating_queue.py

```diff
@@ -68,47 +68,33 @@ def commit(self) -> None:
 
     def get(self) -> BatchedOperations:
         start = time.process_time()
```

Comment:
I don't think it's ok to use this function for the purpose of measuring time -- it's not wall clock time

Reply:
hmm, from what I see, start is only used in `logger.debug`

while variables
```
            t0 = time.monotonic()
            t1 = time.monotonic()
            wait_remaining -= t1 - t0
```
are the ones that are actually used to implement the timeout

so it should be safe and it's only a matter of what information we want to log?

Reply:
It's safe then, but the information will not be meaningful, as it measures CPU/user time, not wall clock time. We should use `monotonic` anyway, because it's not affected by system time changes